symplyphysics/laws/gravity/gravity_force_from_mass_and_distance.py

```diff
@@ -23,6 +38,36 @@
     distance_between_mass_centers**2)
 
 
+# Derive law from the gravitational potential energy
+
+potential = gravitational_potential_energy.law.rhs.subs({
+    gravitational_potential_energy.first_mass: first_object_mass,
+    gravitational_potential_energy.second_mass: second_object_mass,
+    gravitational_potential_energy.distance_between_mass_centers: distance_between_mass_centers,
+})
+
+def potential_field_function(point) -> ScalarValue:
+    assert isinstance(point, CartesianPoint)
+    return potential.subs(
+        distance_between_mass_centers, 
+        sqrt(point.x**2 + point.y**2 + point.z**2),
```

Comment:
It seems, this assumption requires adding a condition to the law, that space is 3-dimensional and linear. If space is curved, we cannot apply this distance function.

Reply:
I agree, I'll mention that